hbase-it/src/test/java/org/apache/hadoop/hbase/TestIntegrationTestBase.java

```diff
@@ -40,11 +40,10 @@ public void testMonkeyPropertiesParsing() {
     conf.set(MonkeyConstants.BATCH_RESTART_RS_RATIO, "0.85");
     conf.set(MonkeyConstants.MOVE_REGIONS_MAX_TIME, "60000");
     conf.set("hbase.rootdir", "/foo/bar/baz");
-
     final Properties props = new Properties();
-    IntegrationTestBase testBase = new IntegrationTestDDLMasterFailover();
     assertEquals(0, props.size());
-    testBase.loadMonkeyProperties(props, conf);
+    new IntegrationTestDDLMasterFailover();
```

Comment:
Why a new without assignment?

Reply:
Because the assignment is flagged as a dead store.

Reply:
Then do we really need to create this instance? It has other side effects?